packages/@aws-cdk/aws-redshift/lib/parameter-group.ts

```diff
@@ -0,0 +1,96 @@
+import { Construct, IResource, Resource } from '@aws-cdk/core';
+import { CfnClusterParameterGroup } from './redshift.generated';
+
+/**
+ * Possible Parameter Group Families
+ * used for defining {@link ClusterParameterGroupProps.family}.
+ * > At this time, redshift-1.0 is the only version of the Amazon Redshift engine.
+ * see https://docs.aws.amazon.com/redshift/latest/mgmt/working-with-parameter-groups.html
+ */
+export enum ParameterGroupFamily {
+  REDSHIFT_1_0 = 'redshift-1.0',
+  DUMMY = 'dummy', // hack to circumvent known bug see https://github.com/aws/aws-cdk/pull/6948#issuecomment-604015109
```

Comment:
Can we just not have this enum at all, since it only has one value? When a second value comes along, then we can introduce it.

Reply:
Do you mean to just use a constant? Or hardcode the family?

I have made it a constant and the property a string, which I check during runtime, throwing an error, when anything else is passed.

Reply:
I mean:

* Remove the `ParameterGroupFamily` enum.
* Remove the `family` property in `ClusterParameterGroupProps`.
* Always pass `'redshift-1.0'` to `CfnClusterParameterGroup`'s ` `parameterGroupFamily` property inside the implementation of `ClusterParameterGroup`.